src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -171,12 +187,13 @@ Status CoreWorkerPlasmaStoreProvider::FetchAndGetFromPlasmaStore(
     absl::flat_hash_map<ObjectID, std::shared_ptr<RayObject>> *results,
     bool *got_exception) {
   const auto owner_addresses = reference_counter_->GetOwnerAddresses(batch_ids);
-  RAY_RETURN_NOT_OK(
-      raylet_client_->FetchOrReconstruct(batch_ids,
-                                         owner_addresses,
-                                         fetch_only,
-                                         /*mark_worker_blocked*/ !in_direct_call,
-                                         task_id));
+  // TODO this IPC needs to be skipped in shared mode
+  //  RAY_RETURN_NOT_OK(
+  //      raylet_client_->FetchOrReconstruct(batch_ids,
+  //                                         owner_addresses,
+  //                                         fetch_only,
+  //                                         /*mark_worker_blocked*/ !in_direct_call,
+  //                                         task_id));
```

Comment:
Should we propagate the mutable flag in for the get?

Reply:
Oops, yeah, missed this...